db/migrate/20191007161920_backfill_paths_for_backend_api_configs.rb

```diff
@@ -0,0 +1,12 @@
+class BackfillPathsForBackendApiConfigs < ActiveRecord::Migration
+  disable_ddl_transaction!
+
+  def change
+    query = System::Database.oracle? ? BackendApiConfig.where('path is NULL') : BackendApiConfig.where(path: '')
```

Comment:
We should avoid DML in Rails migrations as they are not tested and tend to break the upgrade path of onprem customers.